### PR TITLE
Rename log files as seen by users

### PR DESCRIPTION
**Pull request by lucafavatella**
~~Requires deployment changes so it cannot be merged yet.~~ Confirmed with @dincho - it can be merged.

This is a PR with basic code and doc changes for log files.
Delivers https://www.pivotaltracker.com/story/show/163319939

TODO:
* [x] Rebase and retarget on minerva branch.
* [x] Release notes.
* [x] User docs.
  * I see no conflicts of concurrent changes with https://github.com/aeternity/aeternity/pull/2126
* [x] Wiki.
  * [Updated](https://github.com/aeternity/aeternity/wiki/Troubleshooting/_compare/6dc261ce7436d4e613d7832f4c196e1b6755899d...d04868f4d334fdbe3ba3ea47e428fc65f032da9a).

### CODE REVIEW

**Comment by lucafavatella** [comment]
Rebased on minerva branch.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -11,21 +11,21 @@
...
-  
-  
+
 ## Logs, error output, etc.
- 
- (`epoch.yaml` configuration file, `log/epoch.log` log file plus other related logs, e.g., `log/epoch_sync.log`, `log/epoch_mining.log`; if it’s long, please paste to https://ghostbin.com/ and insert the link here)
- 
- ...
 
+*`epoch.yaml` configuration file*
```
Comment (zp-sd) [inline ISSUE_TEMPLATE.md]: `epoch.yaml` 🤔

**Comment by lucafavatella** [comment]
Smoke system tests job [failed on CI](https://circleci.com/gh/aeternity/aeternity/5019):
```
%%% aest_sync_SUITE ==> new_node_joins_network: FAILED
%%% aest_sync_SUITE ==> {timeout,[{aest_nodes,wait_for_value,6,
                      [{file,"/home/circleci/aeternity/_build/system_test+test/extras/system_test/helpers/aest_nodes.erl"},
                       {line,799}]},
          {aest_nodes,loop_for_values,6,
                      [{file,"/home/circleci/aeternity/_build/system_test+test/extras/system_test/helpers/aest_nodes.erl"},
                       {line,761}]},
          {aest_sync_SUITE,new_node_joins_network,1,
                           [{file,"/home/circleci/aeternity/_build/system_test+test/extras/system_test/aest_sync_SUITE.erl"},
                            {line,240}]},
          {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1539}]},
          {test_server,run_test_case_eval1,6,
                       [{file,"test_server.erl"},{line,1055}]},
          {test_server,run_test_case_eval,9,
                       [{file,"test_server.erl"},{line,987}]}]}
```

Unrelated to this PR.

I see already [tracked](https://www.pivotaltracker.com/story/show/163093465).

I am taking the liberty to restart the CI then.

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (lucafavatella) [reply ISSUE_TEMPLATE.md]: https://www.pivotaltracker.com/story/show/163792443

As I expect we fix in master too, I kept it here as it shall be fixed in both master and minerva branches.

Comment (dincho) [reply ISSUE_TEMPLATE.md]: epoch.yaml or aeternity.yaml ?

Comment (lucafavatella) [reply ISSUE_TEMPLATE.md]: https://github.com/aeternity/aeternity/pull/2136

**Pull request merged**